### PR TITLE
[27586] Guided tour for user onboarding (Backlogs)

### PR DESCRIPTION
**Pull request by HDinger**
This adds a class needed for the onboarding tour implemented in https://github.com/opf/openproject/pull/6743. Further a test case is added.

### CODE REVIEW

File: app/helpers/rb_master_backlogs_helper.rb [hunk 1]
```diff
@@ -122,10 +122,12 @@ def sprint_backlog_menu_items_for(backlog)
     items = {}
 
     items[:task_board] = link_to(l(:label_task_board),
-                                 controller: '/rb_taskboards',
+                                 {controller: '/rb_taskboards',
```
Comment (wielinde) [inline app/helpers/rb_master_backlogs_helper.rb]: I believe that CodeClimate would tell you to align the keys of that hasch

File: app/assets/javascripts/backlogs/burndown.js [hunk 2]
```diff
@@ -63,8 +63,13 @@ RB.Burndown = (function ($) {
...
 
-      var url = RB.urlFor('show_burndown_chart', { sprint_id: $(this).data('this').sprintId, project_id: RB.constants.project_id});
-      window.open(url);
+      var url = RB.urlFor('show_burndown_chart', {
+          sprint_id: $(this).data('this').sprintId,
+          project_id: RB.constants.project_id
+      });
+      
+      var burndownWindow = window.open(url);
+      burndownWindow.sessionStorage.clear();
```
Comment (wielinde) [inline app/assets/javascripts/backlogs/burndown.js]: TODO: I don't get what this line is for as I am not deep enough into this plugin. Is this line still needed?

**Review by wielinde — COMMENTED** [review]
Apparently the Spec needs to be adopted to the new texts and their order. The PR looks fine so far.  Please address the two comments I added to the code.

File: app/assets/javascripts/backlogs/burndown.js (hunk 2 above)
Comment (HDinger) [reply app/assets/javascripts/backlogs/burndown.js]: Good one, it is not need any more 👍

**Pull request merged**